Components/build.gradle.kts

```diff
@@ -53,15 +54,17 @@ if (!singleSet)  {
             val ext =  (gradle as ExtensionAware).extra
             getByName("${ext["ios_secondary_arch"]}Main") {
                 dependencies {
-                    implementation(project(":logging", "${ext["ios_secondary_arch"]}Default"))
+                    implementation(project(":base", "${ext["ios_primary_arch"]}Default"))
+                    implementation(project(":logging", "${ext["ios_primary_arch"]}Default"))
                 }
             }
         }
         sourceSets {
             val ext =  (gradle as ExtensionAware).extra
             getByName("iosarm32Main") {
                 dependencies {
-                    implementation(project(":logging", "iosarm32Default"))
+                    implementation(project(":base", "${ext["ios_primary_arch"]}Default"))
```

Comment:
this should by `arm32` instead of `primary`

Reply:
Fixed